### ISSUE TITLE
Oracle XE 18 Docker Image?

### ISSUE THREAD

**Issue by davelindquist-egistix** [oracle/docker-images#1031]
Now that Oracle XE 18 is out, and now that we have a docker image for Oracle Database 18, can we get a docker image for the new XE (18) also?

Thanks!

**Comment by Djelibeybi** [comment]
Yes, it's coming. OpenWorld was last week, and an internal summit is in progress this week, so expect to see it in the next few weeks. There's already a PR open from an external developer for it, in fact.

**Comment by kontrafiktion** [comment]
Why is the issue closed, when the issue is not resolved?

**Comment by Djelibeybi** [comment]
Hmm, not sure why I closed it. Anyway, it's open again. Sorry about that.

**Comment by Djelibeybi** [comment]
No, that's up to @gvenzl. I'm just the Linux guy.
